### ISSUE TITLE
Asking for adding an important feature

### ISSUE THREAD

**Issue by Samo99** [wangqr/Aegisub#106]
Hi, i hope your are doing fine.

Can you please add video panning from this commit too your next release?

https://github.com/Aegisub/Aegisub/pull/207

And if you can make a shortcut for Select lines that will be great. see pic..
![1](https://user-images.githubusercontent.com/12115578/121473827-e2782200-c9cb-11eb-8942-288e590b7783.png)

**Comment by wangqr** [comment]
Iirc the upstream PR was a work-in-progress. I'll check it later when i have time. meanwhile you may try [Ristellise fork](https://github.com/Ristellise/AegisubDC) which claims have merged that PR.

As for shortcut, they can be configured in keyboard preference. The  command name for "select lines" is `tool/line/select`.

**Comment by Samo99** [comment]
> Iirc the upstream PR was a work-in-progress. I'll check it later when i have time. meanwhile you may try [Ristellise fork](https://github.com/Ristellise/AegisubDC) which claims have merged that PR.
> 
> As for shortcut, they can be configured in keyboard preference. The command name for "select lines" is `tool/line/select`.

Yes, Ristellise fork have it i already have his version on my PC but i need it in your fork cause of the Arabic fix you did couple months ago.

And thanks for the guid for the command.

**Comment by PhosCity** [comment]
It's been a long time since the PR was made and I think it is now complete. This feature has become one of the most used feature for a typesetter in fansubbing. Please consider merging it.

Also I cant use Ristellise fork because it is windows only.

**Comment by samoht0** [comment]
Video panning should be pretty much fine tuned in arch1t3cht's "video_panning_option" branch now.

https://github.com/arch1t3cht/Aegisub/commits/video_panning_option

Now there's the Question: Makes merging sense for this classic/stable fork, while the feature is available in arch1t3cht's multi platform bleeding edge fork?
